docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -154,13 +155,13 @@ Elements related to security:
 
 Elements related to domain [startup and shutdown]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md" >}}):
 
-* `serverStartPolicy`: The strategy for [deciding whether to start]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md#starting-and-stopping-servers" >}}) a WebLogic Server instance. Legal values are ADMIN_ONLY, NEVER, or IF_NEEDED. Defaults to IF_NEEDED.
+* `serverStartPolicy`: The strategy for [deciding whether to start]({{< relref "/userguide/managing-domains/domain-lifecycle/startup#starting-and-stopping-servers" >}}) a WebLogic Server instance. Legal values are ADMIN_ONLY, NEVER, or IF_NEEDED. Defaults to IF_NEEDED.
```

Comment:
Does this link (and any other modified links) work with the original 'startup.md' instead of 'startup'?  I assume this may have been done for a reason that might not reproduce on your particular version of hugo.

Reply:
I'll check the relative cross-references on my machine.

Reply:
I made the changes because the original links (using startup.md) did not work for me. In addition, I looked around and noticed that many other links do NOT have ".md" part. It seems that the links that are in all _index.md work with the ".md" part when they are cross-referenced.